src/BaseBuilder.js

```diff
@@ -112,7 +83,7 @@ class BaseBuilder {
 	/**
 	* @description Add must_not boolean queries
 	* @param {string} operation - Type of query to perform
-	* @param {TERMS|TEXT} field - Field to apply the query to
+	* @param {TERMS|TEXT|Object} field - Field to apply the query to
```

Comment:
Yes, but this actually reminds me I need to update the typedef in the constants file so this works. You can define enums and then use them as params in jsdoc comments.